run_first.py

```
@@ -0,0 +1,181 @@
+    """Check if setup files exist and provide guidance"""
+    root_dir = Path(__file__).parent
+    setup_md = root_dir / "SETUP.md"
+    setup_sh = root_dir / "setup.sh" 
+    setup_ps1 = root_dir / "setup.ps1"
+    
+    print("\n📋 Available setup options:")
+    
+    if setup_md.exists():
+        print(f"  ✅ Setup documentation: {setup_md}")
+    else:
+        print(f"  ❌ SETUP.md not found")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)